build.sbt

```diff
@@ -67,7 +66,9 @@ lazy val commonSettings = Seq(
     val packageFile = (Compile / packageBin).value
     generateClasspathFile(
       targetDir = packageFile.getParentFile,
-      classpath = (Runtime / dependencyClasspath).value)
+      // Also include the jar being built (packageFile) in the classpath
+      classpath = (Runtime / dependencyClasspath).value :+ Attributed.blank(packageFile)
```

Comment:
add comment saying that this is needed for the controlModels and serverModel projects since they are marked as provided.